app-misc/toilet/toilet-0.3~git.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Free replacement for the FIGlet utility"
+DESCRIPTION="Free replacement for the FIGlet utility \
+- a program for making large letters out of ordinary text."
+HOMEPAGE="https://github.com/cacalabs/toilet"
+COPYRIGHT="2004 Sam Hocevar"
+LICENSE="WTFPL"
+REVISION="1"
+srcGitRev=3eb9d58037afb0a1baec6dca82caa045fb2217c0
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="a16b3a6217852d4ec5e9fa94c5600b900710d652910ac041bf1c5c61524a03d4"
+SOURCE_FILENAME="toilet-$portVersion~$srcGitRev.tar.gz"
```

Comment:
Oops, you left a tilde between `$portVersion` and `$srcGitRev`. Could you please replace it with a `-` ?
```diff
-SOURCE_FILENAME="toilet-$portVersion~$srcGitRev.tar.gz"
+SOURCE_FILENAME="toilet-$portVersion-$srcGitRev.tar.gz"
```
(Since there is already a tilde in `$portVersion`, and this is good, we don't really need a second one in `$SOURCE_FILENAME`.)
This change would be nice in order to have a name for the tarball, `toilet-0.3~git-3eb9d58037afb0a1baec6dca82caa045fb2217c0.tar.gz`,  that is consistent with that of the tarballs of other recipes which also end with `~git.recipe`.

Thanks @tts2k for your excellent work and sorry for being picky for this tiny detail :)

Reply:
Fixed. And i don't mind you being picky though. It's your job after all :)